### DISCUSSION TITLE
Include Videos?

### DISCUSSION THREAD

**Discussion by chefredaktion** [sylhare/Type-on-Strap#353] [Q&A] [answered]
Hey,
i am very new at building websites. This is why i have two questions.
While building my website with this awesome theme, i thought about including a video on my website. So the first question is; is this (easy) possible?
The second question is: Where can i modify the colors that are shown, when the dark theme is applied?

Thanks for all responses :)

**Comment by sylhare** [comment, marked as answer]
Yes you can include video using [html5](https://developer.mozilla.org/fr/docs/Web/HTML/Element/video) with the `<video>` tag which is very straightforward, you can paste it inside your markdown and then it will be rendered.

For the color, you just need to override the [`_variable.sass`](https://github.com/sylhare/Type-on-Strap/blob/master/_sass/base/_variables.scss#L53), look for the `data-theme="dark"`, there's the [theme color](https://github.com/sylhare/Type-on-Strap/blob/master/_sass/base/_variables.scss#L53) and the code [highlighting color](https://github.com/sylhare/Type-on-Strap/blob/master/_sass/base/_variables.scss#L99).

**Comment by chefredaktion** [comment]
Thank you!
Very nice project btw ;)
